src/components/TaskDashboard.tsx

```diff
@@ -25,23 +30,9 @@ function TaskDashboard() {
         className="todo-list stack-large stack-exception"
         aria-labelledby="list-heading"
       >
-        <li className="todo stack-small">
-          <div className="c-cb">
-            <input id="todo-0" type="checkbox" />
-            <label className="todo-label" htmlFor="todo-0">
-              Model Task
-            </label>
-          </div>
-          <div className="btn-group">
-            <button type="button" className="btn">
-              save
-              <span className="visually-hidden" />
-            </button>
-            <button type="button" className="btn btn__danger">
-              Delete <span className="visually-hidden" />
-            </button>
-          </div>
-        </li>
+        {taskList.map((name: string) => (
```

Comment:
Não tem nada errado aqui, mas uma dica é que não precisamos informar o tipo sempre, o TypeScript é capaz de inferir tipos baseado nas operacões.

nesse caso, por estarmos usando um **map numa lista de string**, ele sabe que `name` é uma string, sem vc precisar de informar, por ser a variavel que guarda um valor da lista (string).

Nesse caso podemos remover o tipo se quiser:
```suggestion
        {taskList.map((name) => (
```

Reply:
nao foi resolvido